tests/E2E/scripts/run-nightly-tests.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
```

Comment:
You should add a comment about this file, saying it should be ran only on a google compute instance (Will stop the machie otherwise).